README.md

```diff
@@ -1,70 +1,38 @@
-# Getting Started with Create React App
+# Weather App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is my first project as part of the front-end module of the Manchester Codes software development bootcamp.
 
-## Available Scripts
+In this project I have built a Weather App user interface, developing my skills in React along the way.
 
-In the project directory, you can run:
+This app fetches data from the Manchester Codes weather API based on a user search of UK cities.
 
-### `npm start`
+## Installation
```

Comment:
The order of this section is a bit mixed up. I would:
- remove the blank line in the `git clone` step,
- then add step to `cd` into the repo and install dependencies,
- then add step to run the app. Testing could be put in its own short section